apps/vr-tests-react-components/src/stories/Input.stories.tsx

```diff
@@ -4,6 +4,7 @@ import { storiesOf } from '@storybook/react';
 import { Input } from '@fluentui/react-input';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=25262&commitId=49fea1346fbdeee728162ced5c8277ab12e6c7ad&env=prod)

✅ There was 4 screenshots added, 0 screenshots removed, 1775 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Input Converged.With appearance override.default.chromium.png|0| Added|
|Select Converged.With appearance override.default.chromium.png|0| Added|
|SpinButton Converged.With appearance override.default.chromium.png|0| Added|
|Textarea Converged.With appearance override.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
Added tests